### ISSUE TITLE
Pet Talent Window (Hunter) is inoperable

### ISSUE THREAD

**Issue by Mitch3llO01** [Lethay/TalentedClassic#86]
Pet talents cannot be spent in any way, despite attempting to spend the points after enabling (check box) the edit button.
You also cannot create a pet talent template.

Talented v220907

**Comment by Lethay** [comment]
Do you get any error codes? I was able to create and edit pet talent templates just fine, but I haven't tried to spend talent points with an actual pet (I will test this).
What language is your client?

**Comment by Mitch3llO01** [comment]
> Do you get any error codes? I was able to create and edit pet talent templates just fine, but I haven't tried to spend talent points with an actual pet (I will test this). What language is your client?

No error codes or I would provide them. If this changes I will of course update this post.

Tried with both a cunning (Wind Serpent) pet and a Ferocity (Devilsaur) pet. In both cases I had to disable Talented addon in order to spend talents. Interestingly there seems to be a Blizzard issue as well. Disabling Elvui and Talented with a cunnin pet, the second ability Dive as well as the 2 talent ability chained directly below it remain grayed out even though the talents do get spent. They also show 1/1 and 2/2 respectfully while remaining grayed out.

Client language is English.

**Comment by Fernando-Marquardt** [comment]
Yeah, can confirm templates can be created no problem, but the talents can't spend the actual points.

**Comment by Lethay** [comment]
I will give this a look when I get a chance, but life is particularly busy - you may need to hold onto your bowstrings for a few days

**Comment by Fernando-Marquardt** [comment]
No problem, but something I observed, not sure if it is related. On my pet talents, it says there is 13 unspent talent points, but as lvl 23 pet, it should have only 1 point, which is spent already.

I may open another issue about this if that would be better.
